src/patternfly/components/Page/page.scss

```diff
@@ -228,6 +228,7 @@
 
     @media screen and (min-width: $pf-global--breakpoint--lg) {
       display: none;
+      visibility: hidden;
```

Comment:
Looking at this one, I'm a little confused about what's happening. It looks like this class is used to hide the kebab in the example below at larger screen sizes:

![image](https://user-images.githubusercontent.com/21063328/53985695-52f7b080-40ea-11e9-83a5-c91059b16f6a.png)


And at larger screen sizes, it disappears and I see two icons instead. But is there a reason why a similar class isn't defined for these icons? Instead, these icons have `.pf-screen-reader` applied which makes them not visible, but they're still there for screen readers. But aren't the actions associated with these icons included in the kebab (i.e. can we just make them `display:none; visibility:hidden`?
![image](https://user-images.githubusercontent.com/21063328/53985733-702c7f00-40ea-11e9-8e91-03cda7efa5a9.png)

Not sure if this is in the scope of this PR or not (and probably not if that change would break our react implementation), but wanted to mention it.

Reply:
I captured this here https://github.com/patternfly/patternfly-next/issues/1538

Reply:
Good pick up! I didn't see that. I wonder if its any other components?